integration/spark/app/src/test/java/io/openlineage/spark/agent/SparkContainerIntegrationTest.java

```diff
@@ -312,4 +312,16 @@ void testRddWithParquet() {
 
     verifyEvents(mockServerClient, "pysparkRDDWithParquetComplete.json");
   }
+
+  @Test
+  void testMetricsEmit() {
```

Comment:
Would it make sense to rename it to `testDebugFacet`? 
It could be useful to add other assertions related to debug Facet within the same test in the future.

Reply:
Done